edge/pkg/metamanager/process.go

```diff
@@ -109,7 +130,19 @@ func resourceUnchanged(resType string, resKey string, content []byte) bool {
 }
 
 func (m *metaManager) processInsert(message model.Message) {
+	resKey, resType, _ := parseResource(message.GetResource())
```

Comment:
what does "res" stand for in these variables? probably "response"? 

maybe just use key and type as name instead. "res" doesn't add too much values to the names.

Reply:
actually, "res" stands for "resource" here. This short form has been used throughout this file . Let's just keep it as is here for code consistency.